src/charts/line.js

```diff
@@ -596,10 +596,13 @@ define(function(require){
             const normalizedDataByTopic = dataByTopic.reduce((accum, topic) => {
                 let {dates, ...restProps} = topic;
 
-                let newDates = dates.map(d => ({
-                       date: new Date(d[dateLabel]),
-                       value: +d[valueLabel]
-                }));
+                const acceptNullValue = (value) => value === null ? null : +value;
```

Comment:
Oh, sorry, could you move this out of the function and to the top of the file?
That way we won't be recreating the function on each step of the reduce loop.

Reply:
sure